### PR TITLE
Absolutevote : add refreshReputation function

### PR DESCRIPTION
**Pull request by orenyodfat**
- Add refreshReputation function can be called by one to refresh the reputation votes for a given proposal and list of voters during voting process. 

- Correct vesting and upgrade schemes permission at test and test migration script.

### CODE REVIEW

File: contracts/VotingMachines/AbsoluteVote.sol [hunk 1]
```diff
@@ -232,6 +234,37 @@ contract AbsoluteVote is IntVoteInterface {
...
+     */
+    function refreshReputation(bytes32 _proposalId, address[] _voters) public returns(bool) {
+        Proposal storage proposal = proposals[_proposalId];
+        Parameters memory params = parameters[proposal.paramsHash];
+
+        for (uint i = 0; i < _voters.length; i++) {
+            Voter storage voter = proposal.voters[_voters[i]];
+             //check that the voters already votes.
+            if (voter.reputation > 0) {
+                //if the reputation change..so update.
```
Comment (leviadam) [inline contracts/VotingMachines/AbsoluteVote.sol]: Fix comment.

File: contracts/VotingMachines/AbsoluteVote.sol [hunk 2]
```diff
@@ -232,6 +234,37 @@ contract AbsoluteVote is IntVoteInterface {
...
+                if (rep > voter.reputation) {
+                    proposal.votes[voter.vote] = proposal.votes[voter.vote].add(rep - voter.reputation);
+                    proposal.totalVotes = (proposal.totalVotes).add(rep - voter.reputation);
+                  } else if (rep < voter.reputation) {
+                    proposal.votes[voter.vote] = proposal.votes[voter.vote].sub(voter.reputation - rep);
+                    proposal.totalVotes = (proposal.totalVotes).sub(voter.reputation - rep);
+                  }
+                if (rep != voter.reputation) {
+                    voter.reputation = rep;
+                    RefreshReputation(_proposalId,_voters[i],rep);
```
Comment (leviadam) [inline contracts/VotingMachines/AbsoluteVote.sol]: `return true`?

File: contracts/VotingMachines/AbsoluteVote.sol (hunk 1 above)
Comment (orenyodfat) [reply contracts/VotingMachines/AbsoluteVote.sol]: fixed at 3f8f14f

File: contracts/VotingMachines/AbsoluteVote.sol (hunk 2 above)
Comment (orenyodfat) [reply contracts/VotingMachines/AbsoluteVote.sol]: fixed at 3f8f14f

**Pull request merged**